internal/bin/sql/sqlgen.go

```diff
@@ -0,0 +1,371 @@
+// (c) Copyright IBM Corp. 2022
+//go:build go1.18
+
+package main
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	combinations "github.com/mxschmitt/golang-combinations"
+)
+
+const driverStmt = "driver.Stmt"
+
+var _conn_m = map[string]string{}
+var _stmt_m = map[string]string{}
+
+var arrayConn = []string{
+	"driver.Conn",
+	"driver.Execer",
+	"driver.ExecerContext",
+	"driver.Queryer",
+	"driver.QueryerContext",
+	"driver.ConnPrepareContext",
+	"driver.NamedValueChecker",
+}
+
+var arrayStmt = []string{
+	"driver.Stmt",
+	"driver.StmtExecContext",
+	"driver.StmtQueryContext",
+	"driver.NamedValueChecker",
+	"driver.ColumnConverter",
+}
+
+func inArray(el string, arr []string) bool {
```

Comment:
Idea: this function could return the index or -1 if element not found.
Then it serves two purposes in one: if the element exists and it gives you its exact position in the array.
This could be useful, for instance, in the `removeOnceFromArr`. Not sure if it's super useful, so I leave it to you to judge :)

Reply:
Decided to keep it as it is